lib/tasks/mfa_notification.rake

```diff
@@ -0,0 +1,35 @@
+require "resolv"
+
+def mx_exists?(email)
+  domain = email.split("@").last
+  mx_resolver = Resolv::DNS.new
+  mx_resolver.timeouts = 10
+
+  return false if mx_resolver.getresources(domain, Resolv::DNS::Resource::IN::MX).empty?
+  true
+rescue StandardError => e
+  puts "Error during processing: #{$ERROR_INFO}"
+  puts "Backtrace:\n\t#{e.backtrace.join("\n\t")}"
+  false
+end
+
+# login in 2019: rake mfa_notification:send
```

Comment:
> Thanks. We will send mail to users with login in 2019 first and possible after two weeks for rest.

Sounds good. Is there potential to miss a few users in the 2nd batch. E.g.

1. User A last signed-in in 2018.
2. Send 1st batch of emails with `rake mfa_notification:send`. User A doesn't receive this email.
3. User A signs in, their updated_at is now in 2019.
4. Send 2nd batch of emails with `rake mfa_notification:send[2009-01-01,2018-12-31]`. User A doesn't receive email again.

Reply:
Thanks for pointing out. We will use `rake mfa_notification:send[2019-01-01,2019-06-01]` at first and following for next iteration:
```
rake mfa_notification:send[2009-01-01,2018-12-31]
rake mfa_notification:send[2019-06-02]
```
Some users will get notified twice, which shouldn't be a huge deal.